docs/api.rst

```diff
@@ -3981,8 +3989,11 @@ AuditLogDiff
 
         :type: Dict[:class:`int`, :class:`ApplicationCommandPermissions`]
 
-.. this is currently missing the following keys: reason and application_id
```

Comment:
are we still missing reason?

Reply:
there's no audit log change key for `reason`, and the entry reason is documented: https://docs.disnake.dev/en/latest/api.html#disnake.AuditLogEntry.reason